manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -324,24 +343,36 @@ An example for a tox-setup can be found in https://github.com/collective/collect
 Text and Bytes
 ~~~~~~~~~~~~~~
 
-This is by far the biggest issue when porting to Python 3. Read https://portingguide.readthedocs.io/en/latest/strings.html to be prepared.
+This is by far the biggest issue when porting to Python 3.
+Read https://portingguide.readthedocs.io/en/latest/strings.html to be prepared.
 
-As a rule of thumb you can assume that in Python 3 everything should be text and only in very rare cases you need to handle bytes.
+As a rule of thumb, you can assume that in Python 3 everything should be text.
+Only in very rare cases you need to handle bytes.
```

Comment:
This is "tricky" wording.
This may 150% technically correct, but in terms or wording, this is tricky.
I would "just" use "in rare cases"

This is IMHO a bit better because by doing so we do not assume knowledge or something else.
If you are not deep into Plone you do not know if that is a rare or a very rare case.

Reply:
The grammar was fixed in 0fac7f2